test.py

```diff
@@ -425,8 +425,8 @@ def test_ignore(self):
         symbols.handle_sym_addrs(
             pathlib.Path("/tmp/thing"), sym_addrs_lines, all_segments
         )
-        assert symbols.spim_context.bannedRangedSymbols[0].start == 16
-        assert symbols.spim_context.bannedRangedSymbols[0].end == 20
+        assert symbols.spim_context.bannedRangedSymbols[0].start == 0x100
```

Comment:
i dont understand this change. Why was it 16 before (was it treating the address as `0x10` instead of `0x100` ? and now it's correct?

One for @AngheloAlf i think?

Reply:
I haven't looked at this test file at all tbh, so I'm not sure what the original person intended.
I'm not sure how it originally passed the test at all